src/main/java/me/gaagjescraft/network/team/manhunt/events/WorldLoadHandler.java

```diff
@@ -0,0 +1,28 @@
+package me.gaagjescraft.network.team.manhunt.events;
+
+import me.gaagjescraft.network.team.manhunt.Manhunt;
+import me.gaagjescraft.network.team.manhunt.games.Game;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerAdvancementDoneEvent;
+
+public class WorldLoadHandler implements Listener {
+
+    private final Manhunt plugin;
+
+    public WorldLoadHandler(Manhunt plugin){
+        this.plugin = plugin;
+    }
+
+    @EventHandler
+    public void LoadWorldAdvancement(PlayerAdvancementDoneEvent event){
```

Comment:
The PlayerAdvancementDoneEvent is not a reliable way to load a world mid-game, because advancements can be disabled server-wide. Besides, this will cause lag during the game which is probably more annoying for the players than for the server to have an extra world loaded.